Yubico.YubiKey/src/Yubico/YubiKey/SmartcardConnection.cs

```diff
@@ -24,7 +24,7 @@
 
 namespace Yubico.YubiKey
 {
-    internal class CcidConnection : IYubiKeyConnection
+    internal class SmartcardConnection : IYubiKeyConnection
```

Comment:
We've been treating "smart card" as two words. So this should be `SmartCardConnection` to remain consistent with the rest of the SDK.

Be aware that Git may not change the name on the filesystem by default now that this filename exists in a commit. There _is_ a way to force Git to change the case of the file... I just don't remember it off the top of my head. It would be nice if we can get Git and the filesystem to reflect the capitalization change too 😄  (I'm OCD like that 😂 )